### ISSUE TITLE
Combining `backup` and `prune` parameters

### ISSUE THREAD

**Issue by i3v** [gabrie30/ghorg#207]
**Describe the bug**
The `backup` flag does not work in combination with `prune` in v1.7.16, because backup results in "_backup" suffix being [added](https://github.com/gabrie30/ghorg/blob/cd74048b107fc049ae19cf215f08b3e9df073931/cmd/clone.go#L522), while `prune` [expects](https://github.com/gabrie30/ghorg/blob/cd74048b107fc049ae19cf215f08b3e9df073931/cmd/clone.go#L670) a folder without such a suffix. Thus, I get the following error message:

```
Scanning for local clones that have been removed on remote...
ghorg.exe : 2022/06/15 03:23:27 open g:\some\path: The system cannot find the file specified.
```
I guess that any feasible combination of parameters on any OS could be broken by adding both these flags.

**Comment by gabrie30** [comment]
Thanks for raising this @i3v! 

The fix should be in 1.7.17 which should be on master if you want to try it out.

**Comment by i3v** [comment]
@gabrie30 , thanks for the quick fix!
I'm still running in a somewhat related issue, but it's no longer related to backup now (just reported in #210).
